grp/classic.gi

```diff
@@ -1915,6 +1915,102 @@ InstallMethod( Omega,
 InstallMethod( Omega,
     [ IsFunction, IsInt, IsPosInt, IsField and IsFinite ],
     { filt, e, d, R } -> OmegaCons( filt, e, d, Size( R ) ) );
+    
+    
+#############################################################################
+##
+#F  WallForm( <form>, <m> ) . . . . . . . . . . . . . compute the wall of <m>
+##
+# Computes the Wall Form of a matrix. The defintition can be found in
+# [Taylor, page 163].
+BindGlobal( "WallForm", function( form, m )
+    local id,  w,  b,  p,  i,  x,  j;
+
+    # first argument should really be something useful
+    id := One( m );
+
+    # compute a base for Image(id-m), use the most stupid algorithm
```

Comment:
But... why? Why not just write `b := BaseMat( id - m );` ? Probably because you want a special such base, namely a subset of the rows of `id - m` with a corresponding `p`... then state so?